### PR TITLE
Added test with send in plan

### LINKED ISSUE #2539: Test send operation inside a plan
Make tests for using send inside a plan, the test will be something like:

```python
@sy.func2plan([th.Size((2,3))])
def plan_double_abs(x):
    x = x.send(bob)
    x = x + x
    x = th.abs(x)
    return x

ptr_result = plan_double_abs(th.zeros(3))
assert type(ptr_result, sy.PointerTensor)
result = ptr_result.get()
assert torch.equal(result, th.tensor([2., 2., 2.]))
```

### PR DESCRIPTION
**Pull request by mccorby**
Test for plans using `send`.

Closes #2539

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -20,3 +20,6 @@ docs/_modules
 .vscode/
 **/pip-wheel-metadata
 
+# IDE specific
```
Comment (mari-linhares) [inline .gitignore]: This is not related to the PR. Can you remove these changes?

File: test/message/test_plan.py [hunk 2]
```diff
@@ -8,14 +8,10 @@
 from syft.serde.serde import deserialize
 from syft.serde.serde import serialize
 from syft import messaging
-import time
```
Comment (mari-linhares) [inline test/message/test_plan.py]: Thanks for the cleanup!

File: .gitignore (hunk 1 above)
Comment (mccorby) [reply .gitignore]: Done

File: test/message/test_plan.py (hunk 2 above)
Comment (iamtrask) [reply test/message/test_plan.py]: +1
